test_scripts/RC/Capabilities/012_Sent_capabilities_if_HMI_doesnt_respond_to_RC_IsReady_and_responds_to_RC_GetCapabilities_RADIO.lua

```diff
@@ -0,0 +1,84 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/1
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/detailed_info_GetSystemCapability.md
+-- Item: Use Case 1: Alternative flow 2
+--
+-- Requirement summary:
+-- [SDL_RC] Capabilities
+--
+-- Description:
+-- In case:
+-- HMI didn't respond on RC.IsReady request from SDL
+-- and SDL send RC.GetCapabilities request to HMI
+-- and HMI respond on this request with capabilities
+--
+-- SDL must:
+-- Use these capabiltites during ignition cycle
+-- Process RC-related RPCs accordingly
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonRC = require('test_scripts/RC/commonRC')
+
+--[[ Local Variables ]]
+local disabledModule = "CLIMATE"
+local enabledModule = "RADIO"
+
+--[[ Local Functions ]]
+local function getHMIParams()
+  local function getHMICapsParams()
+    if enabledModule == "CLIMATE" then
+      return { commonRC.DEFAULT, nil, commonRC.DEFAULT }
+    elseif enabledModule == "RADIO" then
+      return { nil, commonRC.DEFAULT, commonRC.DEFAULT }
+    end
+  end
+  local hmiCaps = commonRC.buildHmiRcCapabilities(unpack(getHMICapsParams()))
+  hmiCaps.RC.IsReady = nil
+  local buttonCaps = hmiCaps.RC.GetCapabilities.params.remoteControlCapability.buttonCapabilities
+  local buttonId = commonRC.getButtonIdByName(buttonCaps, commonRC.getButtonNameByModule(disabledModule))
+  table.remove(buttonCaps, buttonId)
+  return hmiCaps
+end
+
+local function rpcUnsupportedResource(pModuleType, pRPC, self)
+  local pAppId = 1
+  local mobSession = commonRC.getMobileSession(self, pAppId)
+  local cid = mobSession:SendRPC(commonRC.getAppEventName(pRPC), commonRC.getAppRequestParams(pRPC, pModuleType))
+  EXPECT_HMICALL(commonRC.getHMIEventName(pRPC), {}):Times(0)
+  mobSession:ExpectResponse(cid, { success = false, resultCode = "UNSUPPORTED_RESOURCE" })
+end
+
+local function rpcSuccess(pModuleType, pRPC, self)
+  local pAppId = 1
+  local mobSession = commonRC.getMobileSession(self, pAppId)
+  local cid = mobSession:SendRPC(commonRC.getAppEventName(pRPC), commonRC.getAppRequestParams(pRPC, pModuleType))
+  EXPECT_HMICALL(commonRC.getHMIEventName(pRPC), commonRC.getHMIRequestParams(pRPC, pModuleType, pAppId))
+  :Do(function(_, data)
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", commonRC.getHMIResponseParams(pRPC, pModuleType))
+    end)
+  mobSession:ExpectResponse(cid, commonRC.getAppResponseParams(pRPC, true, "SUCCESS", pModuleType))
+end
```

Comment:
These function have been copied enough that they can probably be included in the `commonRC` file